apps/home/pages/api/avatar/[address].ts

```diff
@@ -42,12 +53,18 @@ function handleSendFile(
 }
 
 async function address(req: NextApiRequest, res: NextApiResponse) {
+  const defaultAvatarUrl = `${HOME_URL}/avatar/${currentDefaultAvatar}.png`
```

Comment:
If the default avatar is in the public directory, you can directly request `/public/avatar/normal.png`